lib/improver/tests/threshold/test_BasicThreshold.py

```diff
@@ -552,10 +540,28 @@ def test_threshold_point_nan(self):
         self.cube.data[0][2][2] = np.NAN
         msg = "NaN detected in input cube data"
         plugin = Threshold(
-            2.0, fuzzy_factor=self.fuzzy_factor, threshold_method='<')
+            2.0, fuzzy_factor=self.fuzzy_factor, inequality='<')
         with self.assertRaisesRegex(ValueError, msg):
             plugin.process(self.cube)
 
+
+class Test__init__(IrisTest):
+
+    """Test error-raising behaviours unique to the init method and the private
+    function _decode_inequality_string."""
+
+    def test_equal_fuzzy_threshold_raises_error(self):
+        """Test that combining fuzziness and equality raises expected error."""
+        threshold = 0.6
+        fuzzy_factor = 0.2
+        inequality = 'LE'
+        msg = ('Inequality method "LE" must exclude equality when using fuzzy '
+               'thresholds')
+        with self.assertRaisesRegex(ValueError, msg):
+            str(Threshold(threshold,
```

Comment:
I think the `str` is not necessary here, as the exception will be raised anyway.

Reply:
Removed.